### PR TITLE
update

### PR DESCRIPTION
**Pull request by kherel**

### CODE REVIEW

File: app/.env.example [hunk 1]
```diff
@@ -18,3 +18,8 @@ API_TEMP_KEY=YOUR_KEY_HERE
 
 #Plausible
 PLAUSIBLE_URL=uz.dersu.assistant.pilot.develop
+
+#Auth0
+AUTH0_DOMAIN=dersu-develop.eu.auth0.com
```
Comment (juandelgado) [inline app/.env.example:23]: We can use here `develop-login.dersu.uz` right?

Comment (kherel) [reply app/.env.example]: Yes, I'm using develop-login.dersu.uz in my real .env.

File: app/lib/config/hive_config.dart [hunk 2]
```diff
@@ -11,10 +14,34 @@ class HiveConfig {
...
+      encryptionCipher: HiveAesCipher(encryptionKey),
+    );
+  }
+
+  static Future<Uint8List> getEncryptionKey() async {
+    final FlutterSecureStorage secureStorage = const FlutterSecureStorage();
+    var containsEncryptionKey = await secureStorage.containsKey(key: 'key');
+    if (!containsEncryptionKey) {
+      var key = Hive.generateSecureKey();
+      await secureStorage.write(key: 'key', value: base64UrlEncode(key));
```
Comment (juandelgado) [inline app/lib/config/hive_config.dart:30]: Should we really use `'key'` as the value here? Feels risky to override by mistake somewhere else in the code 6 months down the line. Why not use something descriptive? If we are saving an encryption key then call it... `dersuHiveEncryptionKey`?

Comment (kherel) [reply app/lib/config/hive_config.dart:30]: )) I agree, but this is the only one secret key for now.

Comment (kherel) [reply app/lib/config/hive_config.dart]: done

**Pull request merged**